### PR TITLE
Fix/qa bugs fix

### PR DESCRIPTION
**Pull request by piloshachar14**

### CODE REVIEW

File: apps/client/src/components/category/category.module.css [hunk 1]
```diff
@@ -1,14 +1,31 @@
 .category {
-  border: 2px white solid;
   border-radius: 0.5rem;
   text-align: center;
   display: flex;
   flex-direction: column;
+  overflow: hidden;
+  padding-left: 0.7em;
+  padding-right: 0.7em;
```
Comment (NadavBenAmi) [inline apps/client/src/components/category/category.module.css]: change it to one line

File: apps/client/src/components/criminals/criminals.tsx [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import { useState } from 'react';
 import styles from './criminals.module.css';
 import { Card, Category, Divider } from '../';
-import { BlockTwoTone, ReportGmailerrorredRounded } from '@mui/icons-material';
 import { Tooltip } from 'react-tooltip';
+import { GiPirateFlag, GiPirateGrave } from 'react-icons/gi';
+import { IconContext } from 'react-icons';
```
Comment (NadavBenAmi) [inline apps/client/src/components/criminals/criminals.tsx:6]: combine to one import

File: apps/client/src/components/criminals/criminals.tsx [hunk 3]
```diff
@@ -39,19 +40,19 @@ export const Criminals: React.FC = () => {
...
-              }}
-            />
-            <Tooltip anchorSelect=".button1" place="top">
-              בני עוולה רגילים
-            </Tooltip>
-          </div>
+          <IconContext.Provider value={{ size: '1.4em' }}>
+            <div className={styles.button}>
+              <GiPirateFlag
+                className={`${styles.regularCriminalsIcon} regularCiminalsIcon`}
```
Comment (NadavBenAmi) [inline apps/client/src/components/criminals/criminals.tsx:46]: why did you combine classnames

File: apps/client/src/components/criminals/criminals.tsx [hunk 4]
```diff
@@ -74,34 +75,35 @@ export const Criminals: React.FC = () => {
...
+              </Tooltip>
+            </div>
+          </IconContext.Provider>
+          <div className={styles.criminalsGrid}>
+            {personaNonGratas.map((pesonaNonGrata, index) => (
+              <Card
+                key={index}
+                title={pesonaNonGrata.name}
+                description=":שתיות מפשיעות
           Lorem ipsum dolor sit amet, consectetur adipiscing elit."
```
Comment (NadavBenAmi) [inline apps/client/src/components/criminals/criminals.tsx:94]: what did you wrote here

File: apps/client/src/components/heading/heading.tsx [hunk 5]
```diff
@@ -1,19 +1,25 @@
 import styles from './heading.module.css';
 import LoginIcon from '@mui/icons-material/Login';
 import SettingsIcon from '@mui/icons-material/Settings';
```
Comment (NadavBenAmi) [inline apps/client/src/components/heading/heading.tsx]: same import

File: apps/client/src/components/heading/heading.tsx [hunk 6]
```diff
@@ -35,7 +41,7 @@ export const Heading: React.FC<Props> = ({ title, desc, isLogin }) => {
         <SettingsIcon
           sx={{
             position: 'sticky',
-            color: '#80FFFF',
+            color: '#808080',
```
Comment (NadavBenAmi) [inline apps/client/src/components/heading/heading.tsx]: it be better to save this color globally, even though you use it only here

File: apps/client/src/components/heading/heading.tsx [hunk 7]
```diff
@@ -44,7 +50,7 @@ export const Heading: React.FC<Props> = ({ title, desc, isLogin }) => {
             '&:hover': {
               transitionDelay: 1,
               cursor: 'pointer',
-              color: '#008080',
+              color: '#A9A9A9',
```
Comment (NadavBenAmi) [inline apps/client/src/components/heading/heading.tsx]: here too

File: apps/client/src/components/toasts/toasts.module.css [hunk 8]
```diff
@@ -2,45 +2,58 @@
   display: flex;
   flex-direction: column;
   gap: 0.5em;
-  padding: 0 1em;
+  padding-right: 1em;
+  padding-left: 1em;
```
Comment (NadavBenAmi) [inline apps/client/src/components/toasts/toasts.module.css]: you need only one line

**Review by NadavBenAmi — CHANGES_REQUESTED** [review]

File: apps/client/src/components/criminals/criminals.tsx (hunk 2 above)
Comment (piloshachar14) [reply apps/client/src/components/criminals/criminals.tsx:6]: cant do it

**Pull request merged**